hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/LinuxContainerExecutor.java

```diff
@@ -111,6 +113,58 @@
   private LinuxContainerRuntime linuxContainerRuntime;
 
   /**
+   * The container exit code.
+   */
+  public enum LinuxContainerExecutorExitCode {
```

Comment:
Since this is an inner class of LCE, you can safely drop the LCE from the enum name, which will make the subsequent code less messy.